OpenRA.Mods.Cnc/Traits/World/TSEditorResourceLayer.cs

```diff
@@ -42,6 +42,8 @@ public TSEditorResourceLayer(Actor self, TSEditorResourceLayerInfo info)
 
 		bool IsValidVeinNeighbour(CPos cell, CPos neighbour)
 		{
+			var mapRamp = ((IMapElevation)Map).Ramp;
```

Comment:
try the cast, to allows mods to keep using this logic, even if disabled.

Reply:
Not sure what you mean here. Mods can keep using this logic if they want and it is still part of the default functionality found in `DefaultMap`.

Reply:
Likewise here.